### PR TITLE
Use ACM 2.7 for DR testing

### PR DESCRIPTION
**Pull request by prsurve**
Signed-off-by: prsurve <prsurve@redhat.com>

### CODE REVIEW

**Review by mbukatov — COMMENTED** [review]
Quay repo [acm-d / acm-custom-registry](https://quay.io/repository/acm-d/acm-custom-registry?tab=tags) has few builds every week. I guess we will need to update it often until there is an RC build.

Eg. right now. the latest tag is `2.7.0-DOWNSTREAM-2022-11-14-16-53-04` while the PR is proposing `2.7.0-DOWNSTREAM-2022-11-08-22-55-45`.

**Comment by prsurve** [comment]
> Quay repo [acm-d / acm-custom-registry](https://quay.io/repository/acm-d/acm-custom-registry?tab=tags) has few builds every week. I guess we will need to update it often until there is an RC build.
> 
> Eg. right now. the latest tag is `2.7.0-DOWNSTREAM-2022-11-14-16-53-04` while the PR is proposing `2.7.0-DOWNSTREAM-2022-11-08-22-55-45`.

I have update the image to latest downstream tag

**Comment by prsurve** [comment]
./start.sh --watch
* Testing connection
* Using baseDomain: prsurve-rp-hub.qe.rh-ocs.com
* oc CLI Client Version: 4.12.0-ec.5
Find snapshot tags @ https://quay.io/repository/stolostron/acm-custom-registry?tab=tags
Enter SNAPSHOT TAG: (Press ENTER for default: 2.7.0-DOWNSTREAM-2022-11-15-16-49-09)
2.7.0-DOWNSTREAM-2022-11-17-03-53-08
* Applying multicluster-hub-cr values
catalogsource.operators.coreos.com/acm-custom-registry created

#####
Wait for acm-custom-registry to reach running state (4min).
* STATUS: acm-custom-registry-fmppd               0/1     ContainerCreating   0          1s
* STATUS: acm-custom-registry-fmppd               0/1     Running   0          6s
* STATUS: acm-custom-registry-fmppd               0/1     Running   0          10s
* acm-custom-registry is running
catalogsource.operators.coreos.com/multiclusterengine-catalog created

#####
Wait for multiclusterengine-catalog to reach running state (4min).
* STATUS: multiclusterengine-catalog-cw4s6        0/1     ContainerCreating   0          1s
* STATUS: multiclusterengine-catalog-cw4s6        0/1     Running   0          6s
* STATUS: multiclusterengine-catalog-cw4s6        0/1     Running   0          10s
* multiclusterengine-catalog is running
* Downstream: true   Release Version: 2.7.0
* Composite Bundle: true   Image Registry (CUSTOM_REGISTRY_REPO): quay.io:443/acm-d
* Using: 2.7.0-DOWNSTREAM-2022-11-17-03-53-08

* Applying SNAPSHOT to multiclusterhub-operator subscription
* Applying CUSTOM_REGISTRY_REPO to multiclusterhub-operator subscription
* Applying SUBSCRIPTION_CHANNEL to multiclusterhub-operator subscription
* Applying MODE to multiclusterhub-operator subscription
* Applying STARTING_CSV to multiclusterhub-operator-subscription (advanced-cluster-management.v2.7.0)

##### Creating the open-cluster-management namespace
namespace/open-cluster-management created

##### Applying prerequisites
serviceaccount/default configured
secret/multiclusterhub-operator-pull-secret unchanged
Warning: resource serviceaccounts/default is missing the kubectl.kubernetes.io/last-applied-configuration annotation which is required by kubectl apply. kubectl apply should only be used on resources created declaratively by either kubectl create --save-config or kubectl apply. The missing annotation will be patched automatically.
serviceaccount/default configured
secret/multiclusterhub-operator-pull-secret created

##### Patching secret with QUAY_TOKEN
secret/multiclusterhub-operator-pull-secret patched (no change)

##### Allow secrets time to propagate #####

##### Applying acm-operator subscription #####
operatorgroup.operators.coreos.com/default created
subscription.operators.coreos.com/acm-operator-subscription created

#####
Wait for multiclusterhub-operator to reach running state (4min).
No resources found in open-cluster-management namespace.
* STATUS: Waiting
No resources found in open-cluster-management namespace.
* STATUS: Waiting
No resources found in open-cluster-management namespace.
* STATUS: Waiting
No resources found in open-cluster-management namespace.
* STATUS: Waiting
No resources found in open-cluster-management namespace.
* STATUS: Waiting
No resources found in open-cluster-management namespace.
* STATUS: Waiting
* STATUS: multiclusterhub-operator-85474cf9f6-vts5d                         0/1     ContainerCreating   0          2s
* STATUS: multiclusterhub-operator-85474cf9f6-vts5d                         0/1     Running             0            6s
* STATUS: multiclusterhub-operator-85474cf9f6-vts5d                         0/1     Running   0            11s
* STATUS: multiclusterhub-operator-85474cf9f6-vts5d                         0/1     Running   0             16s
* multiclusterhub-operator is running

* Beginning deploy...
* Applying the multiclusterhub-operator to install Red Hat Advanced Cluster Management for Kubernetes
multiclusterhub.operator.open-cluster-management.io/multiclusterhub created

#####
Wait for multicluster-operators-application to reach running state (4min).
* STATUS: multicluster-operators-application-994f8dfb6-wxvhd                1/3     Running   0             31s
* multicluster-operators-application is running

File: conf/examples/acm_hub_unreleased_image.yaml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 ---
 ENV_DATA:
-  acm_unreleased_image: "2.5.2-DOWNSTREAM-2022-07-29-15-56-49"
+  acm_unreleased_image: "2.7.0-DOWNSTREAM-2022-11-17-03-53-08"
```
Comment (ebenahar) [inline conf/examples/acm_hub_unreleased_image.yaml:3]: @prsurve I saw your comment that you have updated the image to the latest downstream tag but here it is pointing to a specific image

Comment (prsurve) [reply conf/examples/acm_hub_unreleased_image.yaml:3]: Yes we have to add the downstream tag as there is no latest tags present in the ACM repository

File: ocs_ci/framework/conf/default_config.yaml [hunk 2]
```diff
@@ -210,7 +210,7 @@ ENV_DATA:
   rhel8_template: 'rhel85_ocs4qe'
   # ACM HUB deployment
   deploy_acm_hub_cluster: false
-  default_acm_unreleased_image: "2.5.2-DOWNSTREAM-2022-07-29-15-56-49"
+  default_acm_unreleased_image: "2.7.0-DOWNSTREAM-2022-11-25-10-53-02"
```
Comment (petr-balogh) [inline ocs_ci/framework/conf/default_config.yaml:213]: After conversation with Shylesh I got know that ODF 4.11 should be tested with ACM 2.6, ODF 4.12 with ACM 2.7. Should we move this value out of the default config to the specific ODF version config file? 

So can we move it to here: 
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/ocs_version/ocs-4.12.yaml
?

@keesturam @prsurve WDYT?

Than we have pre defined matrix for ODF version and we will have this defined per version.

Comment (prsurve) [reply ocs_ci/framework/conf/default_config.yaml:213]: Sure we can work on the same to tag specific versions of ACM and submariner with ocs release

**Review by petr-balogh — DISMISSED** [review]

**Comment by prsurve** [comment]
Closing this PR as it's not a valid at this point

**Pull request closed without merging**